style-guide.md

```diff
@@ -159,18 +159,18 @@ sites/all/modules/contrib
 
 When linking to an article in a sentence, use the exact title of the article if possible. If using the exact title, display it in title case. The link should be blue with no other formatting (bold, italics, quotations).   **Example**:
 
-For help with SSH keys, see [Generating SSH Keys](https://www.getpantheon.com/docs/articles/users/generating-ssh-keys/).
+For help with SSH keys, see [Generating SSH Keys](/docs/articles/users/generating-ssh-keys/).
 
 When linking to an article in a sentence without using the exact title, display it as part of the sentence in sentence case. **Example**:
 In WordPress, [advanced custom fields can be exported to code](http://stevegrunwell.com/blog/exploring-the-wordpress-advanced-custom-fields-export-feature/ "Steve Grunwell blog, WordPress post").
 
 When referring to another article to provide detailed instructions that are important for completing the current task, use this format:
 
-For detailed instructions, see [Article Title](www.getpantheon.com).
+For detailed instructions, see [Article Title](/docs/articles/path).
```

Comment:
The links lead to 404s, I think we should wrap the examples
https://github.com/pantheon-systems/documentation/blob/legacy-links/docs/articles/path